README.md

```diff
@@ -1,54 +1,78 @@
+   and add it your `.env` in order to fetch curriculum content
+3. Run `bin/setup`. If your database and GitHub token are configured correctly, this should setup and seed your database, 
+   install dependencies and import content from the curriculum repo. Importing may take a few minutes, depending on your 
+   connection.
+4. Run `bin/test` to run all test suites and linters and ensure installation went smoothly
+5. Run `bin/dev` to start the Rails server and JavaScript bundler
 
-The Odin Project depends on open-source contributions to improve, grow, and thrive. We welcome contributors of all experience levels and backgrounds to help maintain this awesome curriculum and community. If you would like to contribute to our curriculum, be sure to thoroughly read our [contributing guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md).
+#### Useful Tools
+##### Lookbook
+In development, visit `localhost:3000/lookbook` to see a visual collection of our [ViewComponents](https://github.com/ViewComponent/view_component). See [Lookbook](https://github.com/ViewComponent/lookbook) for more info.
 
-## Significant Contributors
+##### Scripts
+- `bin/dev` will run all the processes you need (Rails server, JavaScript bundling and background workers) to host the app locally
+- `bin/test` will run all linting checks and test suites. Useful for checking your work before making a PR
+- `bin/rails db:seed` will set up courses and lessons in the database, and `bin/rails curriculum:update_content` will import it via API. Run automatically during first time setup via `bin/setup`.
 
--   [Erik Trautman](https://github.com/eriktrautman)
--   [Afshin Moktari](https://github.com/afshinator)
--   [Josh Gorchov](https://github.com/gorchov)
--   [Joe Sawyer](https://github.com/zkay)
--   [Bill Walker](https://github.com/mach1010)
--   [Neil Gehani](https://github.com/ngehani)
--   [Ian White](http://github.com/Iawhite76)
--   [Nathan Hall](http://github.com/dominathan)
--   [Abby Jones](http://github.com/AbbyJonesDev)
--   [Kevin Mulhern](https://github.com/KevinMulhern)
--   [Arun Kumar](https://github.com/arku)
--   [Cody Loyd](https://github.com/codyloyd)
--   [Austin Mason](https://github.com/CouchofTomato)
--   [Briggs Elsperger](https://github.com/I3uckwheat)
--   [Rhys](https://github.com/105ron)
--   [L. P. Soai-Van](https://github.com/leosoaivan)
--   [Andy Duss](https://github.com/mindovermiles262)
--   [Ada Chiu](https://github.com/adachiu)
--   [Marvin Gay](https://github.com/marvingay)
--   [Rachel Moser](https://github.com/rlmoser99)
--   [Grace Schwartz](https://github.com/CatQueenCodes)
--   [Tatiana Walton](https://github.com/twalton83)
--   [Valera Kushnir](https://github.com/kashura)
-
----
-
-Created by [Erik Trautman](http://www.github.com/eriktrautman)
+## Significant Contributors
+_This should be kept, but should it be directly in the README or linked to from here?_
```

Comment:
Thoughts? Should this be kept as is? Put into `significant_contributors.md` and linked to? Something else entirely? There's a few spots where this information is duplicated somewhat, especially with the hall of fame

Reply:
We have an idea in the backlog for [creating a team page](https://github.com/TheOdinProject/top-meta/issues/3) which may be a nice replacement/ solution to all this duplication.